salt/metalk8s/addons/nginx-ingress-control-plane/deployed/config-map.sls

```diff
@@ -0,0 +1,29 @@
+{%- set ingress_control_plane_controller_defaults = salt.slsutil.renderer(
+        'salt://metalk8s/addons/nginx-ingress-control-plane/config/ingress-controller.yaml.j2', saltenv=saltenv
+    )
+%}
+
+{%- set ingress_control_plane_controller = salt.metalk8s_service_configuration.get_service_conf(
+        'metalk8s-ingress', 'metalk8s-ingress-control-plane-controller-config', ingress_control_plane_controller_defaults
+    )
+%}
+
+Create Control Plane Ingress Controller configuration Config Map:
+  metalk8s_kubernetes.object_present:
+    - manifest:
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: ingress-nginx-control-plane-controller
+          namespace: metalk8s-ingress
+          labels:
+            app.kubernetes.io/component: controller
+            app.kubernetes.io/instance: ingress-nginx-control-plane
+            app.kubernetes.io/managed-by: salt
+            app.kubernetes.io/name: ingress-nginx
+            app.kubernetes.io/part-of: metalk8s
+            heritage: metalk8s
+          annotations:
+            ingressclass.kubernetes.io/is-default-class: "true"
```

Comment:
Hmmm no :) 

And btw I'm not even sure ingress controller look at this annotation on the configmap, it's only for ingress class (and control-plane is not the default ingress class)